tools/istio-iptables/pkg/cmd/run.go

```diff
@@ -28,6 +30,26 @@ import (
 	dep "istio.io/istio/tools/istio-iptables/pkg/dependencies"
 )
 
+type Runner struct {
+	iptables *builder.IptablesBuilderImpl
+	ext      dep.Dependencies
```

Comment:
I would prefer to replace this with tighter interfaces.
This can be done in another PR, but please add a `// TODO` comment here to track that.
What is great with this PR is that only the special cases remain, so we can better identify the use cases for this interface.

Reply:
Added `TODO`